app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.html

```diff
@@ -116,9 +116,8 @@
         <div class="col-md-7">
           <ui-select ng-model="command.expectedArtifact"
                      class="form-control input-sm">
-            <ui-select-match>{{ basicSettingsCtrl.summarizeExpectedArtifact(command.expectedArtifact) }}
-            </ui-select-match>
-            <ui-select-choices repeat="expected in command.backingData.expectedArtifacts">
+            <ui-select-match>{{  basicSettingsCtrl.summarizeExpectedArtifact($select.selected) }}</ui-select-match>
+            <ui-select-choices repeat="expected.id as expected in command.backingData.expectedArtifacts">
```

Comment:
How are these surfaced? Does it look like uuids or like the summary before?

Reply:
Summary like before:

![image](https://user-images.githubusercontent.com/5456773/32287911-a0dea2aa-bf08-11e7-9121-4ebf2e7444c8.png)